en/flight_modes_fw/takeoff.md

```diff
@@ -8,8 +8,8 @@ Vehicles are [hand or catapult launched](#catapult-hand-launch) by default, but
 
 :::note
 
-- This mode requires a good position estimate (e.g. from GPS).
-- The vehicle must be armed before this mode will activate.
+- This mode requires at least a valid local position estimate (does not require a global position).
+  <!-- https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/commander/ModeUtil/mode_requirements.cpp -->
```

Comment:
> - This mode requires at least a valid local position estimate (does not require a global position).

I want to make sure that it is obvious that "needing at least a valid local position estimate" means "you can fly indoors with a valid local position". I think " (does not require a global position)."  probably captures that,right?

Reply:
Requires local position, does not require global position, yes.